lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -232,6 +233,18 @@ describe('GatherRunner', function() {
       const results = await GatherRunner.run(config.passes, options);
       expect(results.TestedAsMobileDevice).toBe(false);
     });
+
+    it('works when running via DevTools with emulation applied outside of LH', async () => {
```

Comment:
we should probably add a test for `IsMobileHost=true` but `IsMobilePage=false` to at least document what we want to happen regardless of decision on question above.